### DISCUSSION TITLE
J'aimerais consulter la liste des outils utilisés par chaque Startup

### DISCUSSION THREAD

**Discussion by sbenfares** [betagouv/cap#17] [Besoin]
### Besoin

```bash
Pouvoir identifier quelle startup et quel contact je peux solliciter si j'ai des questions sur l'usage d'un outil particulier (Metabase, Crisp, Brevo ou autre).

Lien direct avec https://github.com/betagouv/cap/discussions/11
```

### Solutions envisageables

```bash
Idées de solution : un annuaire dans l'espace membre avec la liste des outils liés à chaque startup.

Problématique de la MAJ de cet annuaire.
```

### Informations complémentaires

- [ ] J'avais déjà entendu ce besoin au sein de la communauté beta.gouv
- [X] Je me porte volontaire pour participer à la mise en place de la future solution choisie

**Comment by sneko** [comment]
Bonjour @sbenfares , on travaille actuellement sur une version alpha d'un annuaire global (au-delà du scope "beta.gouv") pour référencer les cas d'utilisation des produits ainsi que leurs outils utilisés.

L'outil fait une analyse des produits pour extraire ces données, évitant ainsi "de dépendre de la volonté des gens" à remplir les fiches. Si le questionnaire initial t'intéresse : https://tally.so/r/wb9MeL

Tu peux en savoir + sur : https://github.com/betagouv/beta-communaute-support/issues/8#issue-2081521071

Est-ce que cela pourrait correspondre à ta demande ?

**Reply by sbenfares** [reply]
Oui carrément merci !
